infra/vsts_agent_windows.tf

```diff
@@ -83,7 +95,7 @@ Set-StrictMode -Version latest
 $ErrorActionPreference = 'Stop'
 
 # Disable Windows Defender to speed up disk access
-Set-MpPreference -DisableRealtimeMonitoring $true
+Set-MpPreference -DisableRealtimeMonitoring $true${local.w[count.index].extra}
```

Comment:
what does this change do?

Reply:
This inserts lines 23-26 for the "no-spool" cohort, while doing no change at all to the default machines, so they don't get killed prematurely.

Arguably I could have put that on the blank line below.

The "mergeable" version of this PR will get back to just the default group, and this change inlined.

Reply:
Oh I see, I missed the fact that it has a newline at the beginning so it won’t actually affect this command.